local-cli/eject/eject.js

```diff
@@ -74,6 +80,42 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting Up App Icons.')
+    if(!appIconiOS) {
+      appIconiOS = appIcon;
+    }
+    const pictureSizes = [40, 60, 58, 87, 80, 120, 180];
+    const contentPath = 'ios/SomeTest/Images.xcassets/AppIcon.appiconset/Contents.json';
```

Comment:
Oops! that's what I named the project with app.json, I should make that dynamic. Thank you